va-gov/pages/disability/view-disability-payment-history.md

```diff
@@ -4,4 +4,5 @@ title: View Disability Payment History
 order: 10
 spoke: Manage Benefits
 template: detail-page
+href: https://www.ebenefits.va.gov/ebenefits/about/feature?feature=payment-history
```

Comment:
Not sure if I did this right, but I think this document needs to point to the HREF value here.